### ISSUE TITLE
Even more sound questions.

### ISSUE THREAD

**Issue by Js41637** [overtools/OWLib#8]
I know what you're think, not this guy again and you'd be right. Although this is mostly me bring some things to your attention and getting your opinion on them.

Ok so the recent change you did to extract all sounds and hero sounds worked perfectly, I literally cannot thank you enough. I've been crunching lots of data, I run a full `V` sound dump and then another `v` sound dump to make sure I get all sounds. Something to note here is `V` does not do NPC sounds whereas `v` does. Not sure if bug or its just not designed to handle it, either way it doesn't really matter as I use both to get a full library.

Next thing I did was also extract all `0B2` sounds and filter out all the sounds that was already found. Went through all this and took out all the obvious sounds (Athena, Snowball Offensive, Uprising, Halloween, Hollywood Guy, Lucioball) which remarkably only leaves about 200 files that haven't been linked to a hero/npc and some of these can also be safey ignored such as the background sounds like some lady speaking through a PA.

What's left is about 170 sounds that aren't getting detected by the extractors which is

- A couple of Widowmaker lines (for some reason)
- 1 Mei voiceline about astronauts 
- Bunch of Winston gorilla monkey sounds
- Uprising bots (note some are extracted by `v` but apparently a lot are missing)
- Bastion sounds (possibly all related to Uprising Bastion)

Just figured I'd mention it if you wanted to look deeper into how some of these sound files are referenced. You can view all unmapped sound IDs and preview them on my site.
https://overwatch-item-tracker.github.io/sounds/?hero=unknown

**Comment by naomichan** [comment]
V cannot do NPC sounds because there's no skin data.

**Comment by Js41637** [comment]
Yeah I figured that was the case but other `v` (lowercase) can do it so that'a not a problem.

**Comment by naomichan** [comment]
Because the normal v doesn't take skin data and just goes through all the data it can find.

**Comment by Js41637** [comment]
I'm sorry i think you misunderstood, I'm not saying it's an issue because I can still use both commands to get all sounds. I was just mentioning it in my original message just to make sure if you were aware of it or not. Sorry for the confusion. Main thing I'm inquiring about is the next part of my original post.

**Comment by naomichan** [comment]
Yeah I'm looking into it.

**Comment by naomichan** [comment]
I don't know why some of the Widowmaker voice lines aren't extracting. They _should_ but for whatever reason they're not. They're part of her Huntress skins. Since the other skins do sound fine, there's something weird.

The monkey sounds are used during Winston's ult, which are now being exported.

**Comment by Js41637** [comment]
Oh neat.

You think some of these Uprising bot sounds are un-unused? I mean if you listen to them a lot of them have the bots saying weird stuff that I'm pretty sure wasn't used during the event? Perhaps they might be getting used later?

**Comment by naomichan** [comment]
No idea.

**Comment by Js41637** [comment]
Interestingly, these 3 unknown Widowmaker voices  (from her Comtesse skin) seems to have been removed from the 1.11.0.0 PTR. The files appear in the 1.10.0.2 and 1.10.1.2 extracts.

**Comment by Js41637** [comment]
Same goes for the single Mei voiceline, also removed from the 1.11.0.0 PTR which also highlighted a problem with my code as it removed sounds that don't exist from the master list.

**Comment by Js41637** [comment]
Ok so all the previous missing sounds files (apart from Widowmakers) have had their files added back in the recent patch so dunno why they were apparently removed but seeing as Overtool still finds them they either removed them or changed something and reverted it ¯\\(ツ)/¯

**Comment by Js41637** [comment]
Ok so all the previous missing sounds files (apart from Widowmakers) have had their files added back in the recent patch so dunno why they were apparently removed but seeing as Overtool still finds them they either removed them or changed something and reverted it ¯\(ツ)/¯

Also if your interested don't know if this is expected or not but none of the new Dance Emotes sounds were detected by the `V` export. I figured they would be referenced by the heroes and thus exported.

**Comment by Js41637** [comment]
I'm just guessing here but are some Ultimates in anyway different to other sounds? I've noticed that Hanzo's ultimate sound, more specifically the wolf version for his Wolf skin isn't exported, is the dragons/wolves from the ult separate from Hanzo possibly?

Also no Dance emote sounds extract, only wondering because sounds from Highlight intros extract and I don't know if any Emote sounds do as they are pretty hard to identify apart from these dance ones.

**Comment by naomichan** [comment]
There might be a bug in the extraction code, or there's a missing link somewhere I haven't implemented. If you can find the IDs I can look into it.

**Comment by Js41637** [comment]
I'll have a look

**Comment by Js41637** [comment]
```
Bastion Dance Sounds:
000000066AE9
000000066AEA
```
```
Pharah Guitar Sounds:
000000066B56
000000066B58
000000066B59
000000066B5A
000000066B5B
... theres a lot more
```
```
Lucio Jazz Sounds
000000066BD6
000000066BEF
000000066BD5
000000066BD4
000000066BEF
```

Off topic there is some really creepy sounds in here in this new patch, voice is really distorted and sounds like someone saying stuff in slow motion, really eerie...

**Comment by Js41637** [comment]
Let me know if there is anything else you're after.

**Comment by naomichan** [comment]
Fixed with de6d6c29e7c5c30a311426006ec16963ad4e63ba
